src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -874,7 +885,8 @@ private SymbolAndDiagnostics ComputeImplementationAndDiagnosticsForInterfaceMemb
         /// <param name="implementingType">The type implementing the interface property (usually "this").</param>
         /// <param name="diagnostics">Bag to which to add diagnostics.</param>
         /// <returns>The implementing property or null, if there isn't one.</returns>
-        private static Symbol ComputeImplementationForInterfaceMember(Symbol interfaceMember, TypeSymbol implementingType, DiagnosticBag diagnostics)
+        private static Symbol ComputeImplementationForInterfaceMember(Symbol interfaceMember, TypeSymbol implementingType, DiagnosticBag diagnostics,
+                                                                      bool ignoreImplementationInInterfaces, out bool implementationInInterfacesMightChangeResult)
```

Comment:
>bool ignoreImplementationInInterfaces, out bool implementationInInterfacesMightChangeResult [](start = 70, length = 91)

Please document these parameters, with an explanation of what kinds of language contexts necessitate their presence.